src/content/app/tools/blast/views/blast-submission-results/BlastSubmissionResults.scss

```diff
@@ -0,0 +1,38 @@
+@import 'src/styles/common';
+
+.blastSubmissionResultsContainer {
+  max-width: 1860px;
+  padding: 0 30px $global-padding-bottom 60px;
+}
+
+.listedBlastSubmission:not(:last-child) {
+  margin-bottom: 48px;
+}
+
+.sequenceBoxWrapper {
+  padding: 24px 26px; // TODO: the 26px padding looks like a constant asking to be extracted
+  border: 1px solid $grey;
+}
+
+.sequenceBoxWrapper + .sequenceBoxWrapper {
+  border-top: none;
+}
+
+.resultsSummaryRow {
```

Comment:
Why is it a summary?

Reply:
bcs each row shows the summary of each individual jobs in that submission.